docs/jsonapi/errors.md

```diff
@@ -0,0 +1,56 @@
+[Back to Guides](../README.md)
+
+# JSON API Errors
+
+Rendering error documents requires specifying the error serializer(s):
+
+- Serializer:
+  - For a single resource: `serializer: ActiveModel::Serializer::ErrorSerializer`.
+  - For a collection: `serializer: ActiveModel::Serializer::ErrorsSerializer`, `each_serializer: ActiveModel::Serializer::ErrorSerializer`.
+
+The resource **MUST** have a non-empty associated `#errors` object.
+The `errors` object must have a `#messages` method that returns a hash of error name to array of
+descriptions.
```

Comment:
@buren So, the problem is that I'm documenting how AMS works, not JSON API.  I could  add a link to the spec (like in https://github.com/rails-api/active_model_serializers/pull/1004#issuecomment-182936225 ) somewhere in here or to https://github.com/rails-api/active_model_serializers/blob/master/docs/jsonapi/schema.md